### PR TITLE
fix: restore salesloft retry config

### PR DESCRIPTION
**Pull request by lucasmarshall**
This was removed in #1938 for some reason

### CODE REVIEW

File: packages/core/remotes/impl/salesloft/index.ts [hunk 1]
```diff
@@ -186,26 +186,30 @@ class SalesloftClient extends AbstractEngagementRemoteClient {
...
+                }
+              : {
+                  ...DEFAULT_LIST_PARAMS,
+                  page: next ? parseInt(next) : undefined,
+                },
+            headers: this.#headers,
+          });
+          return response.data;
+        },
+        // the rate limit is 600/minute shared among all users of the API, so we should wait longer than the usual 1s just to be safe
```
Comment (tomkit) [inline packages/core/remotes/impl/salesloft/index.ts:210]: is this only for this endpoint or all salesloft endpoints?

if for all, we should create a rate_limit.ts just for salesloft

**Review by tomkit — APPROVED** [review]

File: packages/core/remotes/impl/salesloft/index.ts (hunk 1 above)
Comment (lucasmarshall) [reply packages/core/remotes/impl/salesloft/index.ts:210]: Just for this

Comment (tomkit) [reply packages/core/remotes/impl/salesloft/index.ts:210]: Got it - can we call that out that it's just for this endpoint?

**Comment by tonyxiao** [comment]
Shoot I think this is due to a bad merge on my part https://github.com/supaglue-labs/supaglue/commit/5e78708ec4afd9275a6e96c6949291328e72ee68

Sorry guys! And thank you @lucasmarshall for catching it!

**Pull request merged**